Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -181,16 +184,26 @@ ErrorOr<NonnullRefPtr<Gfx::Typeface>> FontLoader::try_load_font()
         }
     }
 
-    // We don't have the luxury of knowing the MIME type, so we have to try all formats.
+    auto const& encoded_data = resource()->encoded_data();
```

Comment:
Instead of more ad-hocness here, can we use `Web::MimeSniff::Resource::sniff`? With a `SniffingContext` of `Font`

Reply:
sure, changed it to use MimeSniff.